<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>391</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>801</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/Images/Панель.png);</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="HeroImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>120</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="HeroHP">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>390</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Little Trouble Girl BV&quot;;</string>
    </property>
    <property name="text">
     <string>Health:</string>
    </property>
   </widget>
   <widget class="QLabel" name="HeroClass">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Little Trouble Girl BV&quot;;</string>
    </property>
    <property name="text">
     <string>Warrior</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WarriorButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/Images/Warrior.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ArcherButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>490</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/Images/Archer.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="WizardButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/Images/Mag.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PriestButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>490</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/Images/PriestOrig.png);
background-image: url(:/resource/Images/Priest.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="attackButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Little Trouble Girl BV&quot;;
background-image: url(:/resource/Images/Панель.png);</string>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SpellButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/Images/Панель.png);
font: 24pt &quot;Little Trouble Girl BV&quot;;</string>
    </property>
    <property name="text">
     <string>Spell</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="widget1" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>100</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(149, 158, 177);
border: 5px solid rgb(0, 0, 0);


</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="EnemyHP1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>2</y>
           <width>51</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Little Trouble Girl BV&quot;;
background-color: none;
border: 0px solid transparent;</string>
         </property>
         <property name="text">
          <string>HP:</string>
         </property>
        </widget>
        <widget class="QLabel" name="EnemyWeapon1">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>0</y>
           <width>61</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none;
border: 0px solid transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Enemy1">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget2" native="true">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>9</y>
       <width>191</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>100</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(149, 158, 177);
border: 5px solid rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="EnemyHP2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>0</y>
           <width>63</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Little Trouble Girl BV&quot;;
background-color: none;
border: 0px solid transparent;</string>
         </property>
         <property name="text">
          <string>HP:</string>
         </property>
        </widget>
        <widget class="QLabel" name="EnemyWeapon2">
         <property name="geometry">
          <rect>
           <x>102</x>
           <y>0</y>
           <width>61</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none;
border: 0px solid transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Enemy2">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget3" native="true">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>9</y>
       <width>191</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>100</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(149, 158, 177);
border: 5px solid rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="EnemyWeapon3">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>2</y>
           <width>61</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none;
border: 0px solid transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="EnemyHP3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>0</y>
           <width>71</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Little Trouble Girl BV&quot;;
background-color: none;
border: 0px solid transparent;</string>
         </property>
         <property name="text">
          <string>HP:</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Enemy3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget4" native="true">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>191</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>100</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(149, 158, 177);
border: 5px solid rgb(0, 0, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="EnemyHP4">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>2</y>
           <width>63</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Little Trouble Girl BV&quot;;
background-color: none;
border: 0px solid transparent;</string>
         </property>
         <property name="text">
          <string>HP:</string>
         </property>
        </widget>
        <widget class="QLabel" name="EnemyWeapon4">
         <property name="geometry">
          <rect>
           <x>92</x>
           <y>0</y>
           <width>71</width>
           <height>51</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: none;
border: 0px solid transparent;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Enemy4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="SpellBook">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>-40</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/Images/imgonline-com-ua-Resize-qp0XG2ixTiqwKr.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="Skill2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resource/Images/FireBall.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>260</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resource/Images/MeteorRain.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Skill1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resource/Images/IceBlast.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/resource/Images/TimeLeap.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QFrame" name="frame_6">
       <property name="styleSheet">
        <string notr="true">
background-image: url(:/resource/Images/Рамочка.PNG);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>221</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QFrame" name="frame_7">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QWidget" name="verticalLayoutWidget_3">
            <property name="geometry">
             <rect>
              <x>-1</x>
              <y>0</y>
              <width>221</width>
              <height>211</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8"/>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
